src/syscall_table/riscv64.rs

```diff
@@ -0,0 +1,311 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
```

Comment:
Mh, I don't think the new files you authored should have Amazon copyright :sweat_smile:

Reply:
That's generated from `generate_syscall_tables.sh`, which automatically adds these lines 😂

Reply:
> Mh, I don't think the new files you authored should have Amazon copyright 😅

Technically I updated the script that generates this file. IANAL, but I think it's fine to keep that copyright header if the script is under Amazon copyright and my contribution is licensed under the same licenses.